tests/sentry/integrations/slack/test_message_builder.py

```diff
@@ -608,40 +663,45 @@ def test_issue_alert_with_suggested_assignees(self):
 
         # auto assign group
         ProjectOwnership.handle_auto_assignment(self.project.id, event)
-        suspect_commit = {
-            "commit_id": commit.key,
-            "author_email": commit.author.email,
-        }
+        suspect_commit_text = f"Suspect Commit: {commit.key[:6]} by {user2.email}"  # no commit link because there is no PR
```

Comment:
we can get the repo through the commit as well; would it be worth linking to the commit even if there is no PR?

(probably something that can be implemented in a separate PR)

Reply:
I have no idea if that scenario happens 😅 for now let's just leave it w/o linking it